### PR TITLE
Add "Deploy your app" docs page

### PR DESCRIPTION
**Pull request by michiels**
This PR adds initial copy for the "Deploy your app" page.

### CODE REVIEW

File: www/docs/deploy-app.html [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+<div class="highlighter-rouge"><pre class="highlight"><code>$ git push dokku master
+</code></pre>
+</div>
+
+<p>Dokku, the underlying system that Intercity uses to deploy your apps, will now take care of updating your code, running database migrations, do asset compilation and perform a rolling restart of your app.</p>
+
+
+    </div>
+
```
Comment (jvanbaarsen) [inline www/docs/deploy-app.html:126]: Can we drop this whitelines?

Comment (michiels) [reply www/docs/deploy-app.html:126]: If you know the Jekyll setting for this. To be honest, because it's "generated html" I don't really mind.

Comment (jvanbaarsen) [reply www/docs/deploy-app.html:126]: Oh haha xD I was not aware that this was generated :P In that case, leave it 😄

Comment (michiels) [reply www/docs/deploy-app.html:126]: Yeah, still need to figure out how to properly deploy a static site to Dokku without having to pre-generate in the repository :)

**Pull request merged**